syntax = "proto3"; //版本

package mprpc; //聲明代碼所在的包

message RpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 agrs_size = 3;
}